github/github.go

```diff
@@ -1094,3 +1094,13 @@ func Int64(v int64) *int64 { return &v }
 // String is a helper routine that allocates a new string value
 // to store v and returns a pointer to it.
 func String(v string) *string { return &v }
+
+// StringSlice is a helper routine that allocates a new string slice value
+// to store v and returns a pointer to it.
+func StringSlice(v []string) *[]string {
```

Comment:
Per idiomatic Go, we will never want to have a pointer to a slice since a slice is already a by-reference type.
So please remove this new helper method entirely.